x/auth/vesting/client/testutil/suite.go

```diff
@@ -130,3 +130,233 @@ func (s *IntegrationTestSuite) TestNewMsgCreateVestingAccountCmd() {
+			},
+			expectErr:    false,
+			expectedCode: 0,
+			respType:     &sdk.TxResponse{},
+		},
+		{
+			name: "defaultVesting",
+			args: []string{
+				sdk.AccAddress("addr12______________").String(),
+				fmt.Sprintf("--%s=%s", flags.FlagFrom, val.Address),
+				fmt.Sprintf("--%s=%s", cli.FlagLockup, "testdata/periods1.json"),
+				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+				fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+				fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+			},
+			expectErr:    false,
+			expectedCode: 0,
+			respType:     &sdk.TxResponse{},
+		},
+		{
+			name: "merge",
+			args: []string{
+				sdk.AccAddress("addr10______________").String(),
+				fmt.Sprintf("--%s=%s", flags.FlagFrom, val.Address),
+				fmt.Sprintf("--%s=%s", cli.FlagLockup, "testdata/periods1.json"),
+				fmt.Sprintf("--%s=%s", cli.FlagVesting, "testdata/periods1.json"),
+				fmt.Sprintf("--%s=%s", cli.FlagMerge, "true"),
+				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+				fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+				fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+			},
+			expectErr:    false,
+			expectedCode: 0,
+			respType:     &sdk.TxResponse{},
+		},
+		{
+			name: "bad vesting addr",
+			args: []string{
+				"foo",
+			},
+			expectErr: true,
+		},
+		{
+			name: "no files",
+			args: []string{
+				sdk.AccAddress("addr13______________").String(),
+			},
+			expectErr: true,
+		},
+		{
+			name: "bad lockup filename",
+			args: []string{
+				sdk.AccAddress("addr13______________").String(),
+				fmt.Sprintf("--%s=%s", cli.FlagLockup, "testdata/noexist"),
+			},
+			expectErr: true,
+		},
+		{
+			name: "bad lockup json",
+			args: []string{
+				sdk.AccAddress("addr13______________").String(),
+				fmt.Sprintf("--%s=%s", cli.FlagLockup, "testdata/badjson"),
+			},
+			expectErr: true,
+		},
+		{
+			name: "bad lockup periods",
+			args: []string{
+				sdk.AccAddress("addr13______________").String(),
+				fmt.Sprintf("--%s=%s", cli.FlagLockup, "testdata/badperiod.json"),
+			},
+			expectErr: true,
+		},
+		{
+			name: "bad vesting filename",
+			args: []string{
+				sdk.AccAddress("addr13______________").String(),
+				fmt.Sprintf("--%s=%s", cli.FlagVesting, "testdata/noexist"),
+			},
+			expectErr: true,
+		},
+		{
+			name: "bad vesting json",
+			args: []string{
+				sdk.AccAddress("addr13______________").String(),
+				fmt.Sprintf("--%s=%s", cli.FlagVesting, "testdata/badjson"),
+			},
+			expectErr: true,
+		},
+		{
+			name: "bad vesting periods",
+			args: []string{
+				sdk.AccAddress("addr13______________").String(),
+				fmt.Sprintf("--%s=%s", cli.FlagVesting, "testdata/badperiod.json"),
+			},
+			expectErr: true,
+		},
+	} {
```

Comment:
ditto, Why is the test execution block wrapped in parentheses?

Reply:
ditto, for is wrapped around testcases itself!